.gitignore

```diff
@@ -51,83 +51,23 @@ jabref.xml
 
 
 
-# Created by https://www.gitignore.io/api/gradle,java,jabref,intellij,eclipse,netbeans,windows,linux,macos
```

Comment:
What happened to the gitignore? This diff seems strange to me if I think about the purpose of the PR.